applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.json

```diff
@@ -1,6 +1,6 @@
 [
  {
-        "nrf5340_audio_dk_snr": 1000,
+        "nrf5340_audio_dk_snr": 1050161015,
```

Comment:
Remove this change. We do not want to push serial numbers to this file

Reply:
SNR accident committed. Will remove.